crates/sui-indexer-alt-jsonrpc/src/lib.rs

```diff
@@ -185,6 +194,7 @@ impl Default for RpcArgs {
         Self {
             rpc_listen_address: "0.0.0.0:6000".parse().unwrap(),
             max_rpc_connections: 100,
+            fullnode_rpc_url: "http://localhost:9000".to_string(),
```

Comment:
I was thinking about this default -- what are your thoughts on making this flag optional, and only adding the write module if it exists? It gets around the tricky issue of coming up with a default URL.